src/routes/apiRouter.js

```diff
@@ -466,6 +488,7 @@ router.post('/hexes/earnings', postHexEarnings)
 router.get('/hexes/earnings', getHexEarnings)
 router.get('/auth', recaptcha.middleware.verify, getAuth)
 router.get('/cell/hotspots/:id', getCellHotspotData)
+router.get('/cell/hotspots/:id/cells', getCellsForHotspot)
```

Comment:
I was on the fence about just replacing `/cell/hotspots/:id` but I left it for now. That API will eventually be removed

Reply:
ok yeah my first thought was the `/cell/..../cells` sounded a bit redundant, but if there would be some conflicts on the `:id` route then we can stick with this for now